src/Mod/PartDesign/Gui/TaskExtrudeParameters.h

```diff
@@ -72,6 +88,9 @@ class TaskExtrudeParameters : public TaskSketchBasedParameters
         SelectFace,
         SelectShape,
         SelectShapeFaces,
+        SelectFace2,
+        SelectShape2,
+        SelectShapeFaces2,

```

Comment:
I'm totally not sold on that. Maybe to a method that accepts that enum we should simply pass second argument with the side?